### DISCUSSION TITLE
Grafana dashboard available?

### DISCUSSION THREAD

**Discussion by PepsiJH** [galexrt/dellhw_exporter#58] [Q&A] [answered]
is there any beautiful grafana dashboard for use ？

**Comment by galexrt** [comment, marked as answer]
@PepsiJH there is this dashboard available: https://github.com/galexrt/dellhw_exporter/tree/main/contrib/monitoring/grafana-dashboards

It should still work, but it doesn't display certain new metrics such as, e.g., nics, chassis_info.
